Documentation/ContentObjects/UserAndUserInt/_Hostname.php

```diff
@@ -2,18 +2,21 @@
 
 declare(strict_types=1);
 
-namespace Vendor\SitePackage\UserFunctions;
+namespace MyVendor\SitePackage\UserFunctions;
+
+use Psr\Http\Message\ServerRequestInterface;
 
 final class Hostname
 {
     /**
      * Return standard host name for the local machine
      *
-     * @param  string Empty string (no content to process)
-     * @param  array  TypoScript configuration
+     * @param string Empty string (no content to process)
+     * @param array TypoScript configuration
+     * @param ServerRequestInterface $request The current PSR-7 request object
```

Comment:
This comment contains no information beyond what the FQN would give you?

Reply:
It also does not harm, Maybe not everyone knows what a ServerRequestInterface is.

Reply:
We have 2 request objects in TYPO3. The original TYPO3 Request also known als TYPO3_REQUEST and the Extbase Request (which is dying with each further TYPO3 version). What about:
```suggestion
     * @param ServerRequestInterface $request The TYPO3 request object
```